test/unit/services/content_item_retriever_test.rb

```diff
@@ -62,11 +62,15 @@ class ContentItemRetrieverTest < ActiveSupport::TestCase
 
     context "when content store unavailable" do
       should "return empty content item hash" do
-        response = { status: 500, body: {}.to_json }
-        stub_request(:get, @request_url).to_return(response)
-
+        stub_content_store_isnt_available
         assert_equal ContentItemRetriever.fetch(@slug), {}
       end
+
+      should "notifies Sentry" do
+        stub_content_store_isnt_available
+        GovukError.expects(:notify)
+        ContentItemRetriever.fetch(@slug)
+      end
```

Comment:
Should we be adding this expectation to the other tests too?

Reply:
Yes. That makes sense. I've updated the other specs. Thank you